tests/console/velociraptor_client_event_collect.pm

```diff
@@ -49,53 +45,52 @@ sub run {
+    background_script_run "velociraptor-client frontend -v --config server.conf > /dev/null 2>&1 &";
+    assert_script_run "velociraptor-client config client --config server.conf > client.conf";
+
+    # update config file
+    script_output 'yq -i ".Client.max_poll = 1" client.conf';
+    script_output 'yq -i ".Client.max_poll_std = 1" client.conf';
+    script_output 'yq -i ".Client.min_poll = 1" client.conf';
+    script_output 'yq -i ".Client.default_max_wait = 1" client.conf';
+
     # copy config files
-    assert_script_run "cp /root/vrr/config/client.conf  /etc/velociraptor/client.config";
-    assert_script_run "sed -i \"s/sensor-frontend/localhost/g\" /etc/velociraptor/client.config";
+    assert_script_run "cp client.conf  /etc/velociraptor/client.config";
 
     # start client
-    systemctl "start velociraptor-client";
+    sleep 10;
+    systemctl "restart velociraptor-client";
     systemctl "status velociraptor-client";
-
     # check server accessible
-    assert_script_run "curl --insecure --user admin:admin https://localhost:8889/app/index.html";
+    assert_script_run "curl -k https://localhost:8000/server.pem";
 
     # generate server api config
-    assert_script_run "sed -i 's/sensor-frontend/localhost/g' vrr/config/server.conf";
-    assert_script_run "velociraptor-client --config vrr/config/server.conf config api_client --name admin --role administrator api.config.yaml";
+    assert_script_run "velociraptor-client --config server.conf config api_client --name admin --role administrator api.config.yaml";
     # get client info from server
     assert_script_run "velociraptor-client --api_config ~/api.config.yaml query 'SELECT * FROM info()'";
 
     # get client id
     my $clientid = script_output('velociraptor-client --api_config ~/api.config.yaml query \'SELECT *, os_info.hostname as Hostname, client_id FROM clients()\' | grep -oP \'"client_id": "\K.*(?=")\'', 120);
 
-    # install virtualenv
-    assert_script_run "$system_python_version  -m venv vrr-api";
-    assert_script_run "vrr-api/bin/pip install pyvelociraptor";
     # add client monitoring
-    assert_script_run('vrr-api/bin/pyvelociraptor --config api.config.yaml \'SELECT add_client_monitoring(artifact="SUSE.Linux.Events.ExecutableFiles") FROM scope()\'');
-    sleep 90;
+    my @artifacts = qw(SUSE.Linux.Audit.SystemLogins SUSE.Linux.Events.DNS SUSE.Linux.Events.ExecutableFiles SUSE.Linux.Events.ImmutableFile SUSE.Linux.Events.NewFiles SUSE.Linux.Events.NewFilesNoOwner SUSE.Linux.Events.NewHiddenFile SUSE.Linux.Events.NewZeroSizeLogFile SUSE.Linux.Events.Packages SUSE.Linux.Events.ProcessStatuses SUSE.Linux.Events.SSHLogin SUSE.Linux.Events.Services SUSE.Linux.Events.SshAuthorizedKeys SUSE.Linux.Events.SystemLogins SUSE.Linux.Events.TCPConnections SUSE.Linux.Events.Timers SUSE.Linux.Events.UserAccount SUSE.Linux.Events.UserGroupMembershipUpdates SUSE.Linux.Sys.At SUSE.Linux.Syslog.SSHLogin);
```

Comment:
Only event artifacts can be added to the client monitoring table, so `SUSE.Linux.Audit.SystemLogins` `SUSE.Linux.Sys.At` and `SUSE.Linux.Syslog.SSHLogin`  should not be in this list.